maintenance/wikia/setWikiFactoryVariable.php

```diff
@@ -54,22 +55,20 @@ public function execute() {
 			die( "Error: Empty variable name.\n" );
 		}
 
-		if ( $set && $remove ) {
-			die( "Error: Cannot set and remove the variable at the same time\n" );
+		if ( $set && $remove && $append ) {
```

Comment:
Shouldn't it be checked for all combinations?
`$set && $remove` || `$set and $append` || `$remove && $append`